integrated_channels/canvas/client.py

```diff
@@ -44,7 +47,78 @@ def update_content_metadata(self, serialized_data):
     def delete_content_metadata(self, serialized_data):
         pass
 
-    def _get_oauth_access_token(self, client_id, client_secret):
+    def _post(self, url, data, scope):
```

Comment:
do we intend to use scope now? if not. we can add it later when needed

Reply:
unless the interface requires it somehow

Reply:
Whoops, forgot to take this part out when using degreed as a reference. I don't yet see how scope would fit into our canvas integration here, just with how we're having users provide us with their auth credentials. But perhaps it'll come up later and we can re-add.